packages/data-stores/src/site/actions.ts

```diff
@@ -306,6 +306,17 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		} catch ( e ) {}
 	}
 
+	function* setLaunchpadScreenOnSite( siteSlug: string, launchpadScreen: string ) {
```

Comment:
I'm fine adding this. Obviously there's precedent and it's clear. I do think we could also use the existing setSiteSettings() method.